docs/contributing/contributing-rules.md

```diff
@@ -461,6 +461,9 @@ subcategory:
 ##### secure default
 
 A secure default rule makes use of inherently secure libraries, frameworks, configurations, or settings. These rules enforce the mitigation of common security concerns, such as preventing cross-site request forgery (CSRF) by properly verifying inbound requests in Django or Flask applications.
+
+A secure defalt rule contains remediation suggests applying a one-time setting that ensures security throughout the codebase without the need for repeated application by developers. For example, configuring a global security setting in a web application framework that applies to all routes and inputs.
```

Comment:
A secure default rule containing remediation suggests applying a one-time setting that ensures security throughout the codebase without the need for developers to apply it repeatedly. For example, you can configure a global security setting for a web application framework that applies to all routes and inputs.

Reply:
I will reword - note that secure default rules **must** suggest (by our definition) a fix, so it's not "containing remediation" - thanks for the catch.